runtime/syntax/cs.vim

```diff
@@ -190,6 +190,18 @@ syn match	csUnicodeNumber	+\\U00\x\{6}+ contained contains=csUnicodeSpecifier di
 syn match	csUnicodeSpecifier	+\\[uUx]+ contained display
 
 syn region	csString	matchgroup=csQuote start=+"+ end=+"\%(u8\)\=+ end=+$+ extend contains=csSpecialChar,csSpecialError,csUnicodeNumber,@Spell
+
+for s:i in range(3, get(g:, "cs_raw_string_quote_count", 8))
```

Comment:
This `cs_raw_string_quote_count` should be documented.

Reply:
@chrisbra I've added a "CSHARP" section to `syntax.txt` describing the 2 new variables. How does that look to you?

Reply:
thanks